fast search users and hash tags

linhhdinh opened the pull request.

linhhdinh committed:
fast search users and hash tags

nayakravi commented on fast_search.go:
```diff
@@ -0,0 +1,26 @@
+	return search.fastSearch(query, search.tags)
+}
+
+func (search *Search) fastSearch(query string, fn func(string) (*SearchResult, error)) (*SearchResult, error) {
```
I don't see the point of this function. Can you please explain why we cannot simply call the respective functions from the above methods?

linhhdinh committed:
changes per pull request

nayakravi committed:
convert app to trueanthem module

This is so that an import of this module does not end up importing
both this and the parent module.
